practiceapp/src/main/java/com/practiceapp/practiceapp/controller/community/GetPostsController.java

```diff
@@ -0,0 +1,36 @@
+package com.practiceapp.practiceapp.controller.community;
+
+import com.mashape.unirest.http.exceptions.UnirestException;
+import com.practiceapp.practiceapp.entity.CommunityEntity;
+import com.practiceapp.practiceapp.entity.PostEntity;
+import com.practiceapp.practiceapp.service.CommunityService;
+import org.bson.json.JsonObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "/community")
+public class GetPostsController {
+
+    @Autowired
+    private CommunityService communityService;
+
+    @RequestMapping(path = "/posts",method = RequestMethod.GET)
+    public List<PostEntity> getPosts(@RequestParam String communityid){
```

Comment:
Adding communityID as path variable would be better

Reply:
I changed it. Thank you!